addons/GearSwap/libs/ccConfig.lua

```diff
@@ -0,0 +1,30 @@
+-- Jobs you want to execute with, recomment put all active jobs you have lua for will look for <job>.lua or <playername>_<job>.lua files

```

Comment:
I don't know how GS usually does this, but normally user files would go into the `GearSwap/data/` folder. That folder is special in that it won't be updated and overwritten. For example, if you ever wanted to update this file with new default values, the file would be overwritten with the new default file for all users who modified it.

Two options you can do then, either move this file to the aforementioned `data` folder, then it will be downloaded initially but ignored in any future updates (can make a subfolder for cc there as well), or forgo this file entirely. I know you want to show people how to use cc through this file, but I think that would work better if you added a blank file to the `data` folder, and keep this file here as an example, and name it as such (possibly refer to it in a readme or w/e). That way people trying it for the first time without looking at the config file will not experience weird behavior, which they will complain to you (and us) about in the end.